src/NeoSharp.Application/Client/IConsoleHandler.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace NeoSharp.Application.Client
+{
+    public interface IConsoleHandler : IConsoleReader, IConsoleWriter
```

Comment:
I am not a fan of this Reader/Writer idea

Reply:
Could you explain more your thoughts?

Reply:
I like the idea because who going to use this will have a very specialized version of the interface but I understand @osmirnov... having just a interface can work too....

Reply:
1) There are no cases in real life for Read/Write separation. Usually, it is ReadOnly and ReadWrite.
2) I am quite opposite to introduce interfaces a) until it is needed or b) as a protection level from a developer